projects/storefrontlib/src/cms-components/product/product-list/container/product-list-component.service.ts

```diff
@@ -41,7 +39,8 @@ export class ProductListComponentService {
     protected activatedRoute: ActivatedRoute,
     protected currencyService: CurrencyService,
     protected languageService: LanguageService,
-    protected router: Router
+    protected router: Router,
+    protected viewConfig: ViewConfig
```

Comment:
Let's just call it `config` instead of `viewConfig` as it contains all the configurations in our application, and by convention from what I'm seeing